CONTRIBUTING.md

```diff
@@ -123,6 +123,84 @@ infrastructure :
   without interaction. **Agent are not unregistered!**
 
 
+## Setting Up a Hot Standby
+
+You can managed a hot standby Postgres instance with an agent with a custom
+`docker-compose.override.yml`:
+
+``` yaml
+# Contents of docker-compose.override.yml
+
+version: '2.4'  # Matches docker-compose.yml version.
+
+volumes:
+  data2:
+  run2:
+
+services:
+  postgres:
+    image: postgres:13-alpine  # MUST MATCH secondary service image !
+    volumes:
+      - ./docker/setup-replication.sh:/docker-entrypoint-initdb.d/setup-replication.sh
+
+  secondary:
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      PRIMARY_HOST: postgres  # Setting PRIMARY_HOST trigger the standby mode.
+    command: postgres -c shared_preload_libraries=pg_stat_statements
+    volumes:
+      - data2:/var/lib/postgresql/data
+      - run2:/var/run/postgresql
+      - ./share/sql/pg_stat_statements-create-extension.sql:/docker-entrypoint-initdb.d/pg_stat_statements-create-extension.sql
+      - ./docker/setup-replication.sh:/docker-entrypoint-initdb.d/setup-replication.sh
+      com.dnsdock.alias: secondaire.temboard.docker
```

Comment:
This line seems over-indented. Also, this does not work for me (after fixing indentation): which version of docker(-compose) is needed?